extensions/wikia/AdEngine/js/provider/gpt/sourcePointTag.js

```diff
@@ -15,11 +15,10 @@ define('ext.wikia.adEngine.provider.gpt.sourcePointTag', [
 		context = adContext.getContext();
 
 	function SourcePointTag() {
-		GoogleTag.apply(this, arguments);
+		GoogleTag.call(this);
```

Comment:
Basically the issue is caused by the change in line 21. This one is a cosmetic change, isn't it?